### ISSUE TITLE
Allow commands to return arbitrary objects

### ISSUE THREAD

**Issue by juanjoman** [boostercloud/booster#902]
## Description

Currently, the commands are only returning a `true` value meaning that the command was processed correctly. It could become handy to return at least the entity UUID in the response.

## Possible Solution
Allow command handlers to return registered events' information

**Example:**
```typescript
// In this case we only need to return the ID, but I wanted to show a more elaborate example
interface CreatePostResult {
  postID: UUID,
  title: string,
  content: string,
  author: string,
}

@Command({
  authorize: 'all'
})
class CreatePost {
  public constructor(
    readonly title: string, 
    readonly content: string, 
    readonly author: string
  ) {}
  
  public handle(register: Register): PostCreatedResult {
    const postCreatedEvent = new PostCreated(
      UUID.generate(), // We generate the post ID in the server side
      this.title,
      this.content,
      this.author,
    )
    register.events(postCreatedevent)
  }
  return postCreatedEvent // We can return the event here because it matches the defined interface, but we could have returned other object
}
```

**Comment by javiertoledo** [comment]
I've been trying to find a conversation that we had about this topic some time ago about allowing commands to return arbitrary objects (pretty much as any express endpoint) and I still think that's the way to go. I know it breaks the model, but we discussed this for a long time and the conclusion was that it's worth giving the developer that flexibility because in real-world things are not always ideal and you need to do this. A key use case would be to return IDs generated server-side for further tracking, but I doubt we should restrict return objects to IDs exclusively.

Basically what I'm saying and what you're showing in the example is the same, but I would just avoid making assumptions about what a developer can return from a command.

**Comment by NickSeagull** [comment]
Closed in #916
